client/layout/universal-navbar-header/index.jsx

```diff
@@ -1,13 +1,17 @@
 import './nav-style.scss';
 import { useTranslate } from 'i18n-calypso';
 import { useState } from 'react';
+import { useSelector } from 'react-redux';
 import WordPressWordmark from 'calypso/components/wordpress-wordmark';
 import UniversalNavbarBtnMenuItem from 'calypso/layout/universal-navbar-header/btn-menu-item.jsx';
 import UniversalNavbarLiMenuItem from 'calypso/layout/universal-navbar-header/li-menu-item.jsx';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
 
 const UniversalNavbarHeader = () => {
 	const translate = useTranslate();
 	const [ isMobileMenuOpen, setMobileMenuOpen ] = useState( false );
+	const currentRoute = useSelector( getCurrentRoute );
+	const ctaParam = currentRoute.replace( /^\//, '' ) + '-lp';
```

Comment:
This won't behave as expected if we end up loading that component for logged-in users. What's our guarantee that this won't happen?

Reply:
One way to get around it and have the expected predictable result would be to use `getSectionName()` directly and just append `-lp` to it. WDYT?

Reply:
> What's our guarantee that this won't happen?

Probably none. Using the section sounds more future-proof and allows doesn't need the `/` to be removed.
Pushed an update to address this.